.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,30 @@
+## Description
+
+Please include a summary of the change and which issue is fixed or feature developed.
+
+Please also include relevant motivation and context. List any dependencies that are required for this change.
```

Comment:
Probably this line doesn't bring much value as for galleries when you implement new actions, the motivation is that we need it. Either request from customer or from management. Nor we will get much benefit on listings dependencies as we can see them in package.yaml

Reply:
This is not only for implementing new actions, this is for all PRs, that includes when you do updates and changes to the current implemented actions. 

Dependencies mentioned are not related to the package dependencies, but to dependencies on other PRs, other systems or tasks.